test/units/static/JBSplits/TestSetSplitGroupsOf.sol

```diff
@@ -116,31 +116,6 @@ contract TestSetSplitGroupsOf_Local is JBSplitsSetup {
         _splits.setSplitGroupsOf(_projectId, _rulesetId, _splitsGroup);
     }
 
-    function test_GivenProjectIdGtUint56Max() external whenCallerIsController {
```

Comment:
See my above comment about setSplitGroupsOf

Reply:
settled in that comment